src/passes/OptimizeInstructions.cpp

```diff
@@ -2949,19 +2949,21 @@ struct OptimizeInstructions
     if (constant == 0ULL) {
       return walked; // nothing more to do
     }
+    // Add the total constant value we computed to the value remaining here.
+    // Note that if the value is 32 bits then |makeFromInt64| will wrap to 32
+    // bits for us; as all the operations before us, and the add below us, are
+    // adds and subtracts then any overflow is not a problem.
```

Comment:
comma nits for readability:

```suggestion
    // bits for us; as all the operations before us and the add below us are
    // adds and subtracts, any overflow is not a problem.
```